src/nvim/CMakeLists.txt

```diff
@@ -43,6 +44,7 @@ set(GENERATED_UNICODE_TABLES ${GENERATED_DIR}/unicode_tables.generated.h)
 include_directories(${GENERATED_DIR})
 include_directories(${CACHED_GENERATED_DIR})
 include_directories(${GENERATED_INCLUDES_DIR})
+include_directories("../../../utf8proc")
```

Comment:
Yes it is. This PR is not even meant for review yet. I'm just experimenting. @justinmk asked me to do a PR.